docs/dev/web/development/repo-structure.md

```diff
@@ -61,18 +56,16 @@ Namely those packages are
 - `/packages/eslint-config`
 - `/packages/extension-sdk`
 - `/packages/prettier-config`
-- `/packages/tsconfig/`
+- `/packages/tsconfig`
 
 ### OpenCloud Design System
 
 The OpenCloud Design System (`/packages/design-system`) is a collection of components, design tokens and styles which ensure a
 unique and consistent look and feel and branding throughout the OpenCloud Web ecosystem. We hope that you use it, too, so that your
 very own apps and extensions will blend in with all the others. Documentation and code examples can be found in
-the [design system documentation](https://opencloud.design).
+the [design system documentation](https://docs.opencloud.eu/design-system/).
 
-The OpenCloud Design System is a standalone project, but to make development easier we have the code in our mono repo.
-We're planning to publish it on npmjs.com as [@opencloud-eu/design-system](https://www.npmjs.com/package/@opencloud-eu/design-system)
-as soon as possible. Since it's bundled with OpenCloud Web, you should not bundle it with your app or extension.
+The OpenCloud Design System is a standalone project that gets published on npmjs.com as [@opencloud-eu/design-system](https://www.npmjs.com/package/@opencloud-eu/design-system). Since it's bundled with OpenCloud Web, you should not bundle it with your app or extension. However, you can add is as a dev dependency for better IDE support.
```

Comment:
```suggestion
The OpenCloud Design System is a standalone project that gets published on npmjs.com as [@opencloud-eu/design-system](https://www.npmjs.com/package/@opencloud-eu/design-system). Since it's bundled with OpenCloud Web, you should not bundle it with your app or extension. However, you can add it as a dev dependency for better IDE support.
```